apps/snitch_core/lib/core/domain/order/order.ex

```diff
@@ -152,4 +154,30 @@ defmodule Snitch.Domain.Order do
     end)
     |> Money.round(currency_digits: :cash)
   end
+
+  def fetch_image_url(line_item) do
+    image = line_item.product.images |> List.first()
+
+    case UrlValidator.valid_url(image.name) do
+      {:ok, url} ->
```

Comment:
This clause will never hit, as we don't store URL directly.
Alos remove `UrlValidator` module.